packages/glimmer-runtime/lib/dom.ts

```diff
@@ -82,7 +83,7 @@ class DOMHelper {
   }
 
   insertHTMLBefore(parent: HTMLElement, nextSibling: Node, html: string): Bounds {
-    let prev = nextSibling && nextSibling.previousSibling;
```

Comment:
@mixonic Great catch. Can you add a non-svg test for this case? I think something like `<div>foo{{{bar}}}</div>` would reproduce the problem right? (I think the second update will wipe out the "foo" text node.) As you can see, the SVG tests were disabled for quite some time during the refactor. It would be good to notice something like this much earlier in future refactor cycles